src/app/[locale]/layout.tsx

```diff
@@ -21,14 +22,24 @@ export default function LocaleLayout({
   params: { locale: any }
 }) {
   if (!getAllAvailableLocales().includes(locale as any)) notFound()
+  const t = useTranslations('current-organisation')
 
   return (
     <html lang={locale} className={`${font.variable}`}>
       <body className="bg-gray-100">
         <AppProvider>
           <Document className="utrecht-theme">
             <Container className="bg-white">
-              <Header />
+              <Header
+                homepage={{
+                  href: '/',
```

Comment:
Van @savitris begreep ik dat de link in het logo naar de homepage van de gemeente moet gaan, niet naar de homepage van Signalen. En dat het dus ook (voor screenreaders) duidelijk moet zijn naar welke homepage je gaat. Is "homepage" dan voldoende of moet hier ook specifieker staan dat het om de homepage van (in dit geval) Utrecht gaat? En moet er dan ook een melding bij dat je het formulier gaat verlaten?